CHANGELOG.md

```diff
@@ -1,5 +1,87 @@
 ## 6.28.0 (Unreleased)
 
+DEPRECATIONS:
+* compute: deprecated `enable_flow_logs` in favor of `log_config` on `google_compute_subnetwork` resource.  If `log_config` is present, flow logs are enabled, and `enable_flow_logs` can be safely removed. ([#22111](https://github.com/hashicorp/terraform-provider-google/pull/22111))
+* containeranalysis: Deprecated `google_container_registry` resource, and `google_container_registry_image` and `google_container_registry_repository` data sources. Use `google_artifact_registry_repository` instead. ([#22071](https://github.com/hashicorp/terraform-provider-google/pull/22071))
```

Comment:
```suggestion
* containerregistry: Deprecated `google_container_registry` resource, and `google_container_registry_image` and `google_container_registry_repository` data sources. Use `google_artifact_registry_repository` instead. ([#22071](https://github.com/hashicorp/terraform-provider-google/pull/22071))
```

Reply:
registry and analysis are weird in terms of product mapping- gonna call this registry explicitly so folks scanning are more likely to clue in